composer.json

```diff
@@ -12,6 +12,7 @@
     "require": {
         "php": "^7.1 || ^8.0",
         "ext-simplexml": "*",
+        "symfony/form": "^4.0 || ^5.0",
```

Comment:
I think it should be in the require-dev instead

Reply:
right, this must be in require-dev

Reply:
Done.